### DISCUSSION TITLE
How to `followpage` on startpage.com?

### DISCUSSION THREAD

**Discussion by kofm** [tridactyl/tridactyl#4444] [Q&A] [answered]
Some websites (like startpage.com) use form buttons to navigate through pages. Is it still possible to use the followpage cmd in some way? Thanks :)

**Comment by bovine3dom** [comment, marked as answer]
You can just rebind `]]` to a hint mode with a selector that matches the form button exactly with `:bindurl www.startpage.com ]] hint -c [selector matching form button]`.

See https://github.com/tridactyl/tridactyl/blob/master/doc/newsletters/tips-and-tricks/1-hint-css-selectors.md for much more detail than you could possibly want ;)
